graph/oval_graph.py

```diff
@@ -199,10 +203,17 @@ def _get_node_icon(self):
             color=VALUE_TO_COLOR[value],
             icon=VALUE_TO_ICON[icon])
 
+    def get_comment(self):
+        if self.comment is not None:
+            return '<i>' + str(self.comment) + '</i>'
```

Comment:
You have data formatting as part of the `OvalNode`, but you probably want to have it in another layer, so when you change your mind and you want to format the output in a different way, you don't have to modify the core code. In other words, think of how to separate data extraction from the data "styling".